data-model.sql

```diff
@@ -240,22 +252,25 @@ CREATE TABLE audit_event (
 	id bigserial NOT NULL,
 	event_id varchar(100),
 	event_action_code varchar(5),
-	event_date_time timestamp without time zone DEFAULT now(),
+	event_date_time varchar(100),
```

Comment:
Assuming this isn't an actual date/time object? But instead some string representation of one?